docs/LDAP.rst

```diff
@@ -21,61 +21,287 @@ plug to a directory.
+   |                                                | attributes (for example ``department,city``)            |                           |                                    |                            |
+   +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+----------------------------+
+   | exo.ldap.users.attributes.{name}.mapping       | Mapping of the custom user attribute                    |                           | Name of the custom attribute       |                            |
+   +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+----------------------------+
+   | exo.ldap.users.attributes.{name}.type          | Type of the custom user attribute                       | ``text`` or ``binary``    | ``text``                           |                            |
+   +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+----------------------------+
+   | exo.ldap.users.attributes.{name}.isRequired    | Is the attribute required                               | ``true`` or ``false``     | ``false``                          |                            |
+   +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+----------------------------+
+   | exo.ldap.users.attributes.{name}.isMultivalued | Is the attribute multi-valued                           | ``true`` or ``false``     | ``false``                          |                            |
+   +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+----------------------------+
+   | exo.ldap.users.search.scope                    | Scope of the search for users                           | ``base``, ``one``         | ``subtree``                        |                            |
+   |                                                |                                                         | or ``subtree``            |                                    |                            |
+   +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+----------------------------+
+   | exo.ldap.groups.base.dn                        | Semicolon-separated list of full DNs                    | ``true`` or ``false``     | ``ou=groups,dc=company,dc=org``    |                            |
```

Comment:
should we show an example with more than one value ?

Reply:
Yes, done